### ISSUE TITLE
Cannot install PyOMP with pip

### ISSUE THREAD

**Issue by elyurn** [Python-for-HPC/PyOMP#11]
Hello,

When using `pip install numba` and `pip install pyomp`, simply running `import numba.openmp` doesn't work: `ModuleNotFoundError: No module named 'numba.openmp'` (I tested this on both my local linux machine and my univerity cluster).

Do you have any clue?

**Comment by ggeorgakoudis** [comment]
Hi @elyurn, supporting `pip` installation is work-in-progress and the `pyomp` pypi package is a placeholder for now. Instead, we offer installation through `conda`.

**Comment by ggeorgakoudis** [comment]
Did `conda` work for you @elyurn?

**Comment by elyurn** [comment]
Yes, everything works fine with `conda`. Could you maybe please add a warning in the pip package so that people know that it doesn't work yet?

**Comment by ggeorgakoudis** [comment]
Yes, you have a point. We should make this more prominent in the pypi distribution. I will leave this issue open until we fix it.
